modules/storage/cosmos-db/main.bicep

```
@@ -90,8 +114,32 @@ param enableAnalyticalStorage bool = false
 @allowed([ 'FullFidelity', 'WellDefined' ])
 param analyticalStorageSchemaType string = 'WellDefined'
 
-@description('The list of Cassandra keyspaces configurations with tables.')
-param cassandraKeyspaces array = []
+type casandrakeyspace = {
+  enableThroughputAutoScale: bool
+  @maxValue(100000)
+  @minValue(400)
+  throughput: int
+  tables:{}
+  tags: objectOfString
+}
+
+type objectOfString = {
+  *: string
+}
```

```suggestion
type cassandraKeyspace = {
  enableThroughputAutoScale: bool
  @maxValue(100000)
  @minValue(400)
  throughput: int
  tables:{}
  tags: {
    *: string
  }
}
```

can consolidate this since objectOfStrings is made up of primitives.

---

we are [standardizing](https://github.com/Azure/bicep-registry-modules/blob/main/CONTRIBUTING.md#user-defined-types-preview) on 'types' being the declared after outputs.

---

fixed